test/import/FSHImporter.Profile.test.ts

```diff
@@ -1681,6 +1681,58 @@ describe('FSHImporter', () => {
         );
       });
 
+      it('should parse an only rule with a CodeableReference to one type', () => {
+        const input = leftAlign(`
+        Profile: ObservationProfile
+        Parent: Observation
+        * performer only CodeableReference(Practitioner)
+        `);
+
+        const result = importSingleText(input);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(1);
+        assertOnlyRule(profile.rules[0], 'performer', {
+          type: 'Practitioner',
+          isCodeableReference: true
+        });
+      });
+
+      it('should parse an only rule with a CodeableReference to multiple types', () => {
+        const input = leftAlign(`
+        Profile: ObservationProfile
+        Parent: Observation
+        * performer only CodeableReference(Organization or CareTeam)
+        `);
+
+        const result = importSingleText(input);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(1);
+        assertOnlyRule(
+          profile.rules[0],
+          'performer',
+          { type: 'Organization', isCodeableReference: true },
+          { type: 'CareTeam', isCodeableReference: true }
+        );
+      });
+
+      it('should parse an only rule with a CodeableReference to multiple types with whitespace', () => {
+        const input = leftAlign(`
+        Profile: ObservationProfile
+        Parent: Observation
+        * performer only CodeableReference(   Organization    or  CareTeam)
```

Comment:
Just to be sure we're allowing white space in all the places, I'd suggest adding some before the last `)` too:
```suggestion
        * performer only CodeableReference(   Organization    or  CareTeam       )
```

Reply:
Good call!

Reply:
Updated in [9f88612](https://github.com/FHIR/sushi/pull/1292/commits/9f88612248639230232f7cb9157c6dd3bb37663b), but then promptly removed in favor of the tests in `FSHImporter.SDRules.test.ts`.